### DISCUSSION TITLE
Provide a mode for setting the artificial delay as close to 0 as possible while still being able to test certain properties of the IC

### DISCUSSION THREAD

**Discussion by ByronBecker** [dfinity/sdk#3005] [Feature Requests]
I would like to spin up several thousand canisters quickly to run load tests, while also having these tests certain maintain properties of the IC (i.e. canister output queues, message instruction limit & DTS limits, etc.).

In addition, as I continue to build more tests into my project, the artificial delay slows down our integration testing and CI feedback loop.

**Comment by viviveevee** [comment]
I added it to our list for the near future. [Internal ticket](https://dfinity.atlassian.net/browse/SDK-894)

This will mostly be a reversion of #1859

**Comment by viviveevee** [comment]
Completed in #3047, will be part of the 0.14 release
